cmd/kraft/build/build.go

```diff
@@ -127,12 +127,17 @@ func (opts *Build) Run(cmd *cobra.Command, args []string) error {
 	// two independent processtrees if we are using "render" mode (aka the fancy
 	// mode is enabled).
 	if !norender {
+		comps, err := project.Components(ctx)
+		if err != nil {
+			return err
+		}
+
 		// The longest word is "configuring" (which is 11 characters long), plus
 		// additional space characters (2 characters), brackets (2 characters) the
 		// name of the project and the target/plat string (which is variable in
 		// length).
-		for _, targ := range project.Targets() {
-			if newLen := len(targ.Name()) + len(target.TargetPlatArchName(targ)) + 15; newLen > nameWidth {
+		for _, component := range comps {
+			if newLen := len(unikraft.TypeNameVersion(component)) + 15; newLen > nameWidth {
```

Comment:
Can we please add a `const` that describes the value `15`?

```go
const lengthOfBlah = 15  // or equivalent
```

Reply:
(there is a comment about it above so this is more like a nice to have, not a blocking comment)

Reply:
🖖

Reply:
done 💪